### ISSUE TITLE
 You must feed a value for placeholder tensor 'tower_3/resnet_v1_100/Placeholder' with dtype float

### ISSUE THREAD

**Issue by alanMachineLeraning** [auroua/InsightFace_TF#75]
Caused by op 'tower_3/resnet_v1_100/Placeholder', defined at:
  File "/home/zhangweiwei/InsightFace_TF-master/train_nets_mgpu_new.py", line 135, in <module>
    net = get_resnet(images_s[i], args.net_depth, type='ir', w_init=w_init_method, trainable=True, keep_rate=dropout_rate)
  File "/home/zhangweiwei/InsightFace_TF-master/nets/L_Resnet_E_IR_MGPU.py", line 223, in get_resnet
    scope='resnet_v1_%d' % num_layers)
  File "/home/zhangweiwei/InsightFace_TF-master/nets/L_Resnet_E_IR_MGPU.py", line 139, in resnet
    net = tl.layers.DropoutLayer(net, keep=keep_rate, name='E_Dropout')
  File "/home/zhangweiwei/anaconda3/lib/python3.6/site-packages/tensorlayer/layers.py", line 1073, in __init__
    set_keep[name] = tf.placeholder(tf.float32)
  File "/home/zhangweiwei/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/array_ops.py", line 1808, in placeholder
    return gen_array_ops.placeholder(dtype=dtype, shape=shape, name=name)
  File "/home/zhangweiwei/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/gen_array_ops.py", line 4848, in placeholder
    "Placeholder", dtype=dtype, shape=shape, name=name)
  File "/home/zhangweiwei/anaconda3/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/home/zhangweiwei/anaconda3/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3392, in create_op
    op_def=op_def)
  File "/home/zhangweiwei/anaconda3/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1718, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

InvalidArgumentError (see above for traceback): You must feed a value for placeholder tensor 'tower_3/resnet_v1_100/Placeholder' with dtype float
	 [[Node: tower_3/resnet_v1_100/Placeholder = Placeholder[dtype=DT_FLOAT, shape=<unknown>, _device="/job:localhost/replica:0/task:0/device:GPU:3"]()]]
	 [[Node: tower_2/gradients/tower_2/resnet_v1_100/block2/unit_3/bottleneck_v1/conv1_bn1/batchnorm/mul_1_grad/Shape-4-0-VecPermuteNCHWToNHWC-LayoutOptimizer/_8933 = _HostRecv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:2", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_15355...tOptimizer", tensor_type=DT_INT32, _device="/job:localhost/replica:0/task:0/device:GPU:2"]()]]

Process finished with exit code 1
